src/ui/app_view.rs

```diff
@@ -128,11 +128,11 @@ pub fn app_view(
 
 				if key == PhysicalKey::Code(KeyCode::Enter) {
 					{
-						config_search.clone().db.write().add(search_text.get());
-						let _ = config_search.save();
+						env_search.clone().db.add(search_text.get());
+						let _ = env_search.save();
```

Comment:
Wouldn't it be better to just save the db ... since we're not making changes to the config?

Reply:
We could do that later

Reply:
Absolutely. Good catch, can also review these when we go the error handling as well.